src/custom/api/gnosisProtocol/api.ts

```diff
@@ -73,7 +73,7 @@ const DEFAULT_HEADERS = {
   'Content-Type': 'application/json',
   'X-AppId': APP_DATA_HASH.toString(),
 }
-const API_NAME = 'Gnosis Protocol'
+const API_NAME = 'CoW Protocol'
```

Comment:
not sure about this change, maybe it can cause any problem?

Reply:
No problem, it is used for messages.